### PR TITLE
Set up pages for fall speakers

### PR DESCRIPTION
**Pull request by creisle**
Started setting up pages, not sure on the wording but wanted to get the website front-page a little less out of date. We can wait to merge until we have talk titles etc if people prefer though

### CODE REVIEW

**Comment by cmrn-rhi** [comment]
@creisle can you please review and integrate #80, after which I am happy to do the review and merge for this PR.

File: src/archive/2023/2023-09-21.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Sep - Keegan Korthauer
+
+!!! info "Event Details"
+
+    **Date/Time:**
+
+    Thursday, September 21st, 2023 :material-clock: 11:30am - 1:30pm PT
```
Comment (cmrn-rhi) [inline src/archive/2023/2023-09-21.md:7]: Change to 11:00am start

File: src/archive/2023/2023-09-21.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    **Location:**
+
+    :material-map-marker: SFU Big Data Hub, room ASB 10900
+
+    :material-video: UBC, Michael Smith Laboratories, MSL 102 (Live Stream Location)
```
Comment (cmrn-rhi) [inline src/archive/2023/2023-09-21.md:13]: Change to `MSL 101`
We weren't able to book MSL 102 for September.

File: src/archive/2023/2023-09-21.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+<!-- **RSVP:**
+
+If you are interested in attending this seminar *in person*, please fill out [the RSVP form].
+
+**Zoom Meeting:**
```
Comment (cmrn-rhi) [inline src/archive/2023/2023-09-21.md:19]: Remove ZOOM meeting info. We aren't sharing publicly this season, just by special request.

File: src/archive/2023/2023-10-19.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+# Oct - Khanh Dao Duc
+
+!!! info "Event Details"
+
+    **Date/Time:**
+
+    Thursday, October 19th, 2023 :material-clock: 11:30am - 1:30pm PT
```
Comment (cmrn-rhi) [inline src/archive/2023/2023-10-19.md:7]: 11:00am start

File: src/archive/2023/2023-10-19.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+    :material-map-marker:  UBC, Michael Smith Laboratories, MSL 102
+
+    :material-video: SFU Big Data Hub, room ASB 10900 (Live Stream Location)
+
+<!-- **RSVP:**
+
+If you are interested in attending this seminar *in person*, please fill out [the RSVP form].
+
+**Zoom Meeting:**
```
Comment (cmrn-rhi) [inline src/archive/2023/2023-10-19.md:19]: Remove ZOOM meeting section

File: src/archive/2023/2023-11-30.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
+# Nov - Michael Baym
+
+!!! info "Event Details"
+
+    **Date/Time:**
+
+    Thursday, October 19th, 2023 :material-clock: 11:30am - 1:30pm PT
```
Comment (cmrn-rhi) [inline src/archive/2023/2023-11-30.md:7]: 11:00am start

File: src/archive/2023/2023-11-30.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+    :material-map-marker:  UBC, Michael Smith Laboratories, MSL 102
+
+    :material-video: SFU Big Data Hub, room ASB 10900 (Live Stream Location)
+
+<!-- **RSVP:**
+
+If you are interested in attending this seminar *in person*, please fill out [the RSVP form].
+
+**Zoom Meeting:**
```
Comment (cmrn-rhi) [inline src/archive/2023/2023-11-30.md:19]: Remove ZOOM section

File: src/archive/2023/2023-12-14.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
+# Sep - Sabrina Leslie
+
+!!! info "Event Details"
+
+    **Date/Time:**
+
+    Thursday, December 14th, 2023 :material-clock: 11:30am - 1:30pm PT
```
Comment (cmrn-rhi) [inline src/archive/2023/2023-12-14.md:7]: 11:00am start

File: src/archive/2023/2023-12-14.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+    :material-map-marker: SFU Big Data Hub, room ASB 10900
+
+    :material-video: UBC, Michael Smith Laboratories, MSL 102 (Live Stream Location)
+
+<!-- **RSVP:**
+
+If you are interested in attending this seminar *in person*, please fill out [the RSVP form].
+
+**Zoom Meeting:**
```
Comment (cmrn-rhi) [inline src/archive/2023/2023-12-14.md:19]: Remove ZOOM section

File: src/schedule.md [hunk 10]
```diff
@@ -1,55 +1,55 @@
...
+- ### [September 21st, 2023, Thursday](./archive/2023/2023-09-21.md)
 
-    :material-clock: 6:00pm - 9:00pm PT
+    :material-clock: 11:00am - 1:30pm PT
 
-    :material-map-marker:  St. Paul's Hospital: [Cullen Family Lecture Theatre (Room 1477)](https://docs.google.com/document/d/1xHHd14LcrDIZLG7RGBGneLgf12H-FJwpyH7rotQCo9k/edit?usp=sharing)
+    :material-map-marker: SFU, Big Data Hub, room ASB 10900
 
-    Featured Speaker: **Dr. Heng Li**
+    :material-video: UBC, Michael Smith Laboratories, MSL 102 (Live Stream Location)
```
Comment (cmrn-rhi) [inline src/schedule.md:13]: Change MSL 102 to MSL 101.
This change only applies for September.

File: src/schedule.md [hunk 11]
```diff
@@ -1,55 +1,55 @@
...
 
-    Introductory Speaker: **Dr. Melissa Chen (Cara Haney Lab)**
+    Featured Speaker: **Dr. Keegan Korthauer**
 
-- ### [February 16th, 2023, Thursday](./archive/2023/2023-02-16.md)
+    Introductory Speaker: TBA
```
Comment (cmrn-rhi) [inline src/schedule.md:17]: Other sections say "Trainee Speaker:" I think that should be the case here. The only reason it wasn't for the previous schedule is the intro speaker ended up being a PhD and it didn't seem appropriate to label her a "Trainee".

**Review by cmrn-rhi — CHANGES_REQUESTED** [review]
Once these changes are addressed I think we're good to go!

File: src/archive/2023/2023-09-21.md (hunk 1 above)
Comment (creisle) [reply src/archive/2023/2023-09-21.md:7]: fixed

File: src/archive/2023/2023-09-21.md (hunk 2 above)
Comment (creisle) [reply src/archive/2023/2023-09-21.md:13]: fixed

File: src/archive/2023/2023-09-21.md (hunk 3 above)
Comment (creisle) [reply src/archive/2023/2023-09-21.md:19]: done

**Review by cmrn-rhi — APPROVED** [review]

**Pull request merged**